server/data/prisonApiClient.ts

```diff
@@ -16,4 +16,16 @@ export default class PrisonApiClient extends AbstractHmppsRestClient {
   async getUser(user: ServiceUser): Promise<PrisonApiUserDetail> {
     return this.get({ path: '/api/users/me', authToken: user.token })
   }
+
+  async searchActivityLocations(
+    prisonCode: string,
+    date: string,
+    period: string,
+    user: ServiceUser,
+  ): Promise<Location[]> {
+    return this.get({
+      path: `/api/agencies/${prisonCode}/eventLocationsBooked?bookedOnDay=${date}&timeSlot=${period}`,
+      authToken: user.token,
```

Comment:
Likely, do not provide this authToken. Instead pass the user object as a second parameter (see `getInmateDetail`).

Reply:
Had a great discussion around this thats helped me understand the subtleties here - thanks. Weve agreed its probs ok for now.